From 988e9ba3708b5fbef352eb30f3d4c0c2b1ecd719 Mon Sep 17 00:00:00 2001
From: Wright Feng <wright.feng@cypress.com>
Date: Tue, 29 May 2018 02:52:58 -0500
Subject: [PATCH 4701/5242] MLK-19517-08 brcmfmac: fix unused variable
 building warning message

commit  4c1029321a253bfc8f2a24809c3ff7c7b325bbcb from
https://source.codeaurora.org/external/imx/linux-imx.git

The variable "wq_flags" is not used anymore. Remove it.

Signed-off-by: Wright Feng <wright.feng@cypress.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../wireless/broadcom/brcm80211/brcmfmac/sdio.c    |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
index 1de890e..1e78c99 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c
@@ -4424,7 +4424,6 @@ struct brcmf_sdio *brcmf_sdio_probe(struct brcmf_sdio_dev *sdiodev)
 	struct brcmf_sdio *bus;
 	struct workqueue_struct *wq;
 	struct brcmf_fw_request *fwreq;
-	unsigned int wq_flags;
 
 	brcmf_dbg(TRACE, "Enter\n");
 
-- 
1.7.9.5

